sdlf-team/scripts/template-team-repos.yaml

```diff
@@ -150,6 +152,14 @@ Resources:
             Resource: !Ref rSNSTopic
       Topics:
         - !Ref rSNSTopic
+  rSNSTopicSubscription:
+    Type: Custom::SNSSubscription
```

Comment:
`Condition: IsProduction` would also be required for the custom resource, otherwise it would try to subscribe a topic which does not exist

Reply:
Adding this condition. Also, I think that reducing the approvers  to 1 is a good idea.